crates/autopilot/src/infra/persistence/dto/fee_policy.rs

```diff
@@ -2,7 +2,7 @@ use crate::{boundary, domain};
 
 #[derive(Debug, Clone, PartialEq, sqlx::FromRow)]
 pub struct FeePolicy {
-    pub auction_id: domain::AuctionId,
+    pub auction_id: domain::auction::Id,
```

Comment:
nit: I think the sql record struct should have the raw type (not the domain type).

Reply:
Fixed.